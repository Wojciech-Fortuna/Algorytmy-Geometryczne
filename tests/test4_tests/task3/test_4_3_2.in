-0.04271371103102162 0.007908498446146661 0.004302418001236444 -0.03691380547542198
-0.03384274328908614 0.02509599844614667 0.017165321227042896 -0.02848856037738276
-0.011000001353602261 -0.017367236847970993 0.030915321227042908 -0.015345178024441575
0.002528224452849351 -0.03185865841659845 0.02670161154962354 0.0031903611912447033