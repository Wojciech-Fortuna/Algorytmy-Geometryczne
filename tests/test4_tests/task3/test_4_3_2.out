0.005806934807635715 -0.016556444720678153 2 3
0.010025601838031018 -0.020988202988542273 2 4
0.013332591183346312 -0.01619339562692305 3 4